### ISSUE TITLE
Change actionlib namespace of libsoundplay.SoundClient

### ISSUE THREAD

**Issue by garaemon** [ros-drivers/audio_common#119]
`libsoundplay.SoundClient` uses `sound_play` actionlib namespace.
Users cannot use multiple `SoundClient` instances for different sound_play nodes.

https://github.com/ros-drivers/audio_common/blob/master/sound_play/src/sound_play/libsoundplay.py#L117

My proposal is adding the 3rd argument to specify actionlib namespace to `SoundPlay.__init__()` method like:

```python
class SoundClient(object):

    def __init__(self, blocking=False, namespace='sound_play'):
```

**Comment by knorth55** [comment]
@garaemon your proposal seems good! can you make a PR if you have time?

**Comment by knorth55** [comment]
closed by #149 
@garaemon thank you for your contribution!
